CONTRIBUTING.md

```diff
@@ -152,3 +152,54 @@ Getting help with git
 ---------------------
 
 If you're new to ``git``, you'll probably find [gitref.org](http://gitref.org/) helpful.
+
+Setting up a development environment
+------------------------------------
+
+The recommended way to set up a development environment for contributing to scikit-bio is using [Anaconda](https://store.continuum.io/cshop/anaconda/) by Continuum Analytics, with it's associated command line utility `conda`. The primary benefit of `conda` over `pip` is that on some operating systems (ie Linux), `pip` installs packages from source. This can take a very long time to install Numpy, scipy, matplotlib, etc. `conda` installs these packages using pre-built binaries, so the installation is much faster. Another benefit of `conda` is that it provides both package management and virtual environments, which removes the necessity of using `virtualenv` separately. Not all packages are available using `conda`, therefore our strategy is to install as many packages as possible using `conda`, then install any remaining packages using `pip`.
```

Comment:
*provides both package management and virtual environments,* -> *provides both package and environment management,*

To avoid confusion with virtualenv.